### PR TITLE
Feature/rishabh/2 11 2021/UI changes

### PR DESCRIPTION
**Pull request by Ri5ha6h**

### CODE REVIEW

File: app/containers/Dashboard/index.js [hunk 1]
```diff
@@ -6,139 +6,297 @@
...
 import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Menu } from 'antd';
+import { PlusOutlined, MoreOutlined, LoadingOutlined } from '@ant-design/icons';
 
 import { FormattedMessage } from 'react-intl';
 import messages from './messages';
 
-// import './styles.scss';
+import './styles.scss';
```
Comment (meharajM) [inline app/containers/Dashboard/index.js:16]: move all styles of this file from style prop to styles.scss file

File: app/containers/Dashboard/index.js [hunk 2]
```diff
@@ -6,139 +6,297 @@
...
+                            }
+                          >
+                            <SubMenu key="sub1">
+                              <Menu.Item
+                                key="1"
+                                style={{ backgroundColor: 'white' }}
+                              >
+                                <Link to={`/storyline/${item.id}`}>
+                                  <Button
+                                    style={{
```
Comment (meharajM) [inline app/containers/Dashboard/index.js:192]: put all the repeated style code in a const and use it

File: app/containers/Dashboard/index.js [hunk 3]
```diff
@@ -6,139 +6,297 @@
...
+                          backgroundColor: '#f3f4f6',
+                          marginLeft: 6,
+                          borderRadius: 12,
+                        }}
+                      >
+                        <p style={{ fontSize: 16, fontWeight: 600 }}>
+                          LogLine:
+                          <span
+                            style={{
+                              marginLeft: 5,
```
Comment (meharajM) [inline app/containers/Dashboard/index.js:262]: put all the repeated style code in a const and use it, or put it in styles.scss file with a class name and use it

File: app/containers/Profile/index.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            src={User.profileObj.imageUrl}
+          />
+        </div>
+        <div className="profileContent">
+          <h3>{User.profileObj.name}</h3>
+          <h3>{User.profileObj.email}</h3>
+          <div className="profileButton">
+            <div className="driveDiv">
+              <p
+                style={{
```
Comment (meharajM) [inline app/containers/Profile/index.js:43]: move style to styles.scss

File: app/containers/SideBar/index.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+import {
+  UserOutlined,
+  FolderOutlined,
+  PoweroffOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+import Avatar from '../../mppComponents/MppAvatar';
+import Button from '../../mppComponents/MppButton';
+
+import './style.scss';
```
Comment (meharajM) [inline app/containers/SideBar/index.js:22]: move styles of this file into styles.scss

File: app/containers/Stories/index.js [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+import messages from './messages';
+
+export function Stories() {
+  const [stories, setStories] = useState(data);
+  const { SubMenu } = Menu;
+  return (
+    <>
+      <div
+        style={{
```
Comment (meharajM) [inline app/containers/Stories/index.js:30]: move to styles.scss

**Review by meharajM — CHANGES_REQUESTED** [review]

**Review by shashankmotruri — CHANGES_REQUESTED** [review]

File: app/containers/Dashboard/dummyStory.js [hunk 7]
```diff
@@ -0,0 +1,66 @@
+export const data = [
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/dummyStory.js:1]: okay I'll delete it

File: server/controllers/google/google.js [hunk 8]
```diff
@@ -268,5 +262,5 @@ module.exports = {
   listFiles,
   getFile,
   updateFile,
-  deleteFile
+  deleteFile,
 };
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply server/controllers/google/google.js:266]: it's all done by prettyfier plugin

File: app/containers/Dashboard/index.js [hunk 9]
```diff
@@ -50,181 +45,172 @@ export default function Dashboard() {
...
+                              fontSize: 15,
+                              fontWeight: 400,
+                            }}
+                          >
+                            {stories[item].logLine}
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js]: yes they do have null checkers.

File: app/containers/Dashboard/dummyStory.js [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  {
+    sNo: 8,
+    title: 'Freedom',
+    logLine: 'Adam dont know nothing',
+    theme: 'kkkk',
+    genre: 'Action Adventure, None',
+    timeLine: 'old',
+  },
+];
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/dummyStory.js:66]: it's done

File: app/containers/Profile/index.js (hunk 4 above)
Comment (Ri5ha6h) [reply app/containers/Profile/index.js:43]: it's done

File: app/containers/SideBar/index.js (hunk 5 above)
Comment (Ri5ha6h) [reply app/containers/SideBar/index.js:22]: done

File: app/containers/Stories/index.js (hunk 6 above)
Comment (Ri5ha6h) [reply app/containers/Stories/index.js:30]: done

File: app/containers/Dashboard/index.js (hunk 3 above)
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:262]: done

File: app/containers/Dashboard/index.js (hunk 2 above)
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:192]: done

File: app/containers/Dashboard/index.js [hunk 11]
```diff
@@ -6,139 +6,297 @@
...
+        {/* recent stories section */}
+        <div className="recent">
+          <h3 style={{ color: 'black', marginTop: 15 }}>Recent</h3>
+          <Row gutter={[16, 24]}>
+            {recent.map((item, i) => (
+              <Col md={12} lg={8} key={item.sNo}>
+                <Card
+                  title={item.title}
+                  bordered
+                  style={{
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:95]: done

File: app/containers/Dashboard/index.js [hunk 12]
```diff
@@ -6,139 +6,297 @@
...
-                      <h2>Add Story</h2>
-                    </center>
-                  </Card>
-                </Link>
+      <div className="dashboardContent">
+        <Button
+          className="storyButton"
+          shape="round"
+          size="large"
+          style={{ border: 'none' }}
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:74]: done

File: app/containers/Dashboard/index.js [hunk 13]
```diff
@@ -6,139 +6,297 @@
...
+        >
+          <Link className="storyLink" to="/storyline/new">
+            <span className="storySpan" style={{ fontSize: 18 }}>
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:77]: done

File: app/containers/Dashboard/index.js [hunk 14]
```diff
@@ -6,139 +6,297 @@
...
+              Create Story
+            </span>
+            <PlusOutlined
+              className="storyIcon"
+              style={{ fontSize: 20, marginLeft: 10 }}
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:82]: done

File: app/containers/Dashboard/index.js (hunk 1 above)
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:16]: done

File: app/containers/Dashboard/index.js [hunk 15]
```diff
@@ -6,139 +6,297 @@
...
+                  style={{
+                    backgroundColor: '#f3f4f6',
+                    marginLeft: 7,
+                    borderRadius: 12,
+                  }}
+                >
+                  <p style={{ fontSize: 15, fontWeight: 600 }}>
+                    LogLine:
+                    <span
+                      style={{
```
[the remark this answers is not in the document]
Comment (Ri5ha6h) [reply app/containers/Dashboard/index.js:104]: done

**Review by shashankmotruri — APPROVED** [review]

**Pull request merged**